src/components/mobileOnlyAccordion/index.tsx

```diff
@@ -5,7 +5,10 @@ import Accordion from '../accordion';
 
 export type Props = {
   variant?: 'light' | 'dark';
-} & Pick<AccordionProps, 'allowMultiple' | 'allowToggle'>;
+} & Pick<
+  AccordionProps,
+  'allowMultiple' | 'allowToggle' | 'index' | 'onChange'
```

Comment:
unlocks programmatic controls over the accordion so that we can close it when the tenant is selected